backend/nouns/PatchNouns.ts

```diff
@@ -0,0 +1,23 @@
+import { sequelize } from "../DB.js";
+import { router } from "../Router.js";
+
+router.patch("/api/nouns/:nounId", async (req, res) => {
+  const { slug, friendlyName, parentId } = req.body;
+  const toUpdate = {
+    ...(slug !== undefined && { slug }),
```

Comment:
This allows null to go through, was that intended? If not, we could use nullish coalescing operator maybe:

```suggestion
    ...(slug ?? { slug }),
```

Reply:
it was intended, but thinking through it now, I think the only case you'd really want this is for setting `parentId` to null